backend/app/database/falkor/code-graph-backend/api/analyzers/java/analyzer.py

```
@@ -68,34 +81,39 @@ def get_entity_types(self) -> list[str]:
     def add_symbols(self, entity: Entity) -> None:
         if entity.node.type == 'class_declaration':
             interfaces_query = self.language.query("(super_interfaces (type_list (type_identifier) @interface))")
-            interfaces_captures = interfaces_query.captures(entity.node)
+            
+            captures = self._run_query(interfaces_query, entity.node)
+            
             if 'interface' in interfaces_captures:
```

Ruff (0.14.11)

`interfaces_captures` may be undefined, or defined from star imports

(F405)